### DISCUSSION TITLE
No Text

### DISCUSSION THREAD

**Discussion by Amrotu** [Unlucky-Life/ankimon#35] [Ideas]
Hey,

just installed it, looks like an awesome addon! Is it possible to somehow not show the text messages? Find they are disturbing my flow a bit.

**Comment by Unlucky-Life** [comment]
> Hey,
> just installed it, looks like an awesome addon! Is it possible to somehow not show the text messages? Find they are disturbing my flow a bit.

This will be integrated in the next Update ! (Coming end of this week)

**Reply by Amrotu** [reply]
Oh sorry, I'm not that familiar with github, makes way more sense under discussions :D.
Awesome, tyvm!
